File: Lista 1/Exercicio3.java
import java.util.Scanner;

public class NumerosPares {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int numerosPares = 0;

        System.out.println("Digite 10 números inteiros:");

        for (int i = 0; i < 10; i++) {
            System.out.print("Número " + (i + 1) + ": ");
            int numero = scanner.nextInt();
            
            if (numero % 2 == 0) {
                numerosPares++;
            }
        }

        System.out.println("Quantidade de números pares: " + numerosPares);
        
        scanner.close();
    }
}

File: Lista 1/Exercicio3.py
numeros_pares = 0

print("Digite 10 números inteiros:")

for i in range(10):
    numero = int(input(f"Número {i + 1}: "))
    if numero % 2 == 0:
        numeros_pares += 1

print("Quantidade de números pares:", numeros_pares)
